.github/workflows/automerge.yaml

```
@@ -0,0 +1,32 @@
+---
+name: automerge
+"on":
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+  check_suite:
+    types:
+      - completed
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - id: automerge
+        name: automerge
+        uses: "pascalgn/automerge-action@v0.15.5"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          # Don't require a special label to auto merge.
+          MERGE_LABELS: ""
+          MERGE_FILTER_AUTHOR: appstudio-staging-ci
```

I think based on [this logic](https://github.com/pascalgn/automerge-action/blob/eb68b061739cb9d81564f8e812d0b3c45f0fb09a/lib/merge.js#L61) and a [JSON of the pull request](https://api.github.com/repos/hacbs-contract/ec-policies/pulls/279), this should be:

```suggestion
          MERGE_FILTER_AUTHOR: "appstudio-staging-ci[bot]"
```

---

Thanks! Updated.